tests/dvslib/dvs_vlan.py

```diff
@@ -0,0 +1,94 @@
+from dvs_database import DVSDatabase
+
+class DVSVlan(object):
+    def __init__(self, adb, cdb, sdb, cntrdb, appdb):
+        self.asic_db = adb
+        self.config_db = cdb
+        self.state_db = sdb
+        self.counters_db = cntrdb
+        self.app_db = appdb
+
+    def create_vlan(self, vlanID):
+        vlanName = "Vlan{}".format(vlanID)
+        vlan_entry = {"vlanid": vlanID}
+        self.config_db.create_entry("VLAN", vlanName, vlan_entry)
+
+    def remove_vlan(self, vlanID):
+        vlanName = "Vlan{}".format(vlanID)
+        self.config_db.delete_entry("VLAN", vlanName)
+
+    def create_vlan_member(self, vlan, interface, tagging_mode="untagged"):
+        member = "Vlan{}|{}".format(vlan, interface)
+        if tagging_mode:
+            member_entry = {"tagging_mode": tagging_mode}
+        else:
+            member_entry = {"no_tag_mode": ""}
```

Comment:
This is not valid as per YANG model, let me know if it is valid, then I need to change yang model as well.

Reply:
what's the scenario where we use "no_tag_mode". I guess we always have "tagging_mode". It either would be `tagged` or `untagged`. What say?

Reply:
OK, this is used to exercise when the tagging_mode is unspecified. They want to add "no_tag_mode" as a field and empty string as value. And want to ensure that when tagging_mode field is NOT specified, we act as untagged. I don't know why they have done it this way. Instead they could have eliminated the field altogether.But I don't know if that is possible. As these are existing things, I will leave them as is and we can cleanup later.

Reply:
@praveen-li : It is NOT vlaid, so dont add anything to YANG

Reply:
```
       if test_input[0] == "no_tag_mode":
            tagging_mode = None
        else:
            tagging_mode = test_input[1]

```

Reply:
It is better to add Azure PR in such cases, so that it is easy to review. Thx.

Reply:
Sure, https://github.com/Azure/sonic-swss/pull/1242